### PR TITLE
Custom data drift example

### PR DESCRIPTION
**Pull request by riteshkarval**
1. Custom data drift example
2. In cloudevents examples getting the MINIO url from env if running in same.

### CODE REVIEW

File: insurance_custom_drift/data_drift.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+class CustomDrift:
+    def __init__(self):
+        self.features = ['age', 'sex', 'bmi', 'children', 'smoker', 'region', 'charges']
+
+    def load_trainData_ref(self):
+        train_data_file = os.path.join(os.getenv('MM_HOME'), "dc/train.npy")
+        self.train_data = np.load(train_data_file)
+
+    def get_predict_data(self):
```
Comment (songole) [inline insurance_custom_drift/data_drift.py:17]: Can we add some comments here?

File: insurance_custom_drift/data_drift.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+if __name__ == "__main__":
+    mm_config = get_configuration()
+    api = DkubeApi(token=os.getenv("DKUBE_USER_ACCESS_TOKEN"))
+    mm_id = mm_config["envs"]["MM_UUID"]
+    schema_df = api.modelmonitor_schema_to_df(mm_id)
+    custom_drift = CustomDrift()
+    custom_drift.load_trainData_ref()
+    predict_df = custom_drift.get_predict_data()
+    scores = custom_drift.calculate_drift(predict_df.values)
+    predict_data_baseline = baseline_from_schema(predict_df, schema_df)
```
Comment (songole) [inline insurance_custom_drift/data_drift.py]: Using predict data, are we calling this baseline? Does this compute distribution metrics? Should we name accordingly?

File: insurance_custom_drift/dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.7.5
+
+RUN pip3 install pip -U
+
+RUN pip3 install setuptools wheel packaging scikit-learn
+
+RUN pip3 install git+https://github.com/oneconvergence/dkube.git@monitoring-utils
```
Comment (songole) [inline insurance_custom_drift/dockerfile]: shouldn't this branch be 3.7? dkube.git@3.7

File: insurance_custom_drift/readme.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+2. Select **Model Type** as **Regression**
+3. Select **Data type** as **Tabular**
+4. Go to **Drift** and enable drift
+5. Select the algorithm **Custom**.
+6. Provide the previous build image name in **Docker Image** field,
+   - eg: `ocdr/insurance_custom:v1`
+7. Fill start-up command `bash custom.sh`
+8. Submit.
+9.  Go to Schema page and verify schema. 
+10. Close the Schema page, select the model monitor and start.
```
Comment (songole) [inline insurance_custom_drift/readme.md:33]: Can we add some comments about samples files? How to use them what are they for?

File: insurance_custom_drift/data_drift.py (hunk 1 above)
Comment (riteshkarval) [reply insurance_custom_drift/data_drift.py:17]: okay

File: insurance_custom_drift/data_drift.py (hunk 2 above)
Comment (riteshkarval) [reply insurance_custom_drift/data_drift.py]: okay, I will rename.

File: insurance_custom_drift/dockerfile (hunk 3 above)
Comment (riteshkarval) [reply insurance_custom_drift/dockerfile]: yes, I will correct.

File: insurance_custom_drift/readme.md (hunk 4 above)
Comment (riteshkarval) [reply insurance_custom_drift/readme.md:33]: okay

**Pull request merged**